virtcontainers/api.go

```diff
@@ -590,6 +590,10 @@ func statusContainer(sandbox *Sandbox, containerID string) (ContainerStatus, err
 				}
 
 				if !running {
+					virtLog.WithFields(logrus.Fields{
+						"state": container.state.State,
+						"Pid":   container.process.Pid}).
```

Comment:
Nit: I'd make the field name lowercase: `"pid"`.

Reply:
done